index.js

```diff
@@ -109,6 +167,23 @@ function myMap(array, callback) {
 // // => [6,12,18,24]
 // console.log("%c----------", "color: red") 
 
+function myMap(array, callback) {
+  const result = []
+  for (let i = 0; i < array.length; i++) {
+    const element = array[i]
+    const value = callback(element)
+    result.push(value)
+  }
+  return result
+}
+
+function triple(num) {
+  return (num * 3)
+}
```

Comment:
Good! You don't need the parenthesis around the return value. However, with or without the parenthesis, it's good! (: 

```suggestion
function triple(num) {
  return num * 3
}
```